mender-update/v3/update_module.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+#include <mender-update/v3/update_module.hpp>
+
+#include <common/error.hpp>
+#include <common/expected.hpp>
+
+namespace mender {
+namespace update {
+namespace v3 {
```

Comment:
Same here, I propose getting either "module" or "update_module" in between `update` and `v3`.